core/domain/summary_services.py

```diff
@@ -131,3 +131,18 @@ def _generate_query(categories):
         })
 
     return results
+
+
+def get_featured_explorations():
+    """Returns a list of featured explorations."""
+    exp_ids = []
+    exp_ids = exp_services.get_all_exploration_summaries()
```

Comment:
(Btw, it looks like some of the functions in exp_services are misnamed -- they should use 'exploration summaries' instead of 'exp summary dicts'. Be careful of this when you trace through the code.)